installation/sql/postgresql/base.sql

```diff
@@ -275,6 +275,7 @@ INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder",
 (0, 'plg_editors-xtd_module', 'plugin', 'module', 'editors-xtd', 0, 1, 1, 0, 1, '', '', '', 6, 0),
 (0, 'plg_editors-xtd_pagebreak', 'plugin', 'pagebreak', 'editors-xtd', 0, 1, 1, 0, 1, '', '', '', 7, 0),
 (0, 'plg_editors-xtd_readmore', 'plugin', 'readmore', 'editors-xtd', 0, 1, 1, 0, 1, '', '', '', 8, 0),
+(0, 'plg_editors-xtd_newmodule', 'plugin', 'module', 'editors-xtd', 0, 1, 1, 0, 1, '', '', '', 9 0),
```

Comment:
```suggestion
(0, 'plg_editors-xtd_newmodule', 'plugin', 'module', 'editors-xtd', 0, 1, 1, 0, 1, '', '', '', 9, 0),
```
Same as above in my previous review comment for the MySQL file.